lib/Core/ErrorState.h

```diff
@@ -75,7 +79,7 @@ class ErrorState {
   void registerInputError(ref<Expr> error);
 
   void executeStoreSimple(ref<Expr> address, ref<Expr> error,
-                          ref<Expr> valueWithError);
+                          ref<Expr> valueWithError, KInstruction *i);
```

Comment:
@Himeshi As noted above, this could have been `llvm::Instruction *instr` instead.